Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/RectTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+
+namespace Newtonsoft.Json.UnityConverters.Tests.ConvertingUnityTypes
+{
+    public class RectTests : TypeTester<Rect>
+    {
+        public static (Rect deserialized, string serialized)[] representations = new[] {
+            (new Rect(1, 2, 3, 4), @"{""x"":1.0,""y"":2.0,""width"":3.0,""height"":4.0}")
+        };
+
+        protected override void ConfigureSettings(JsonSerializerSettings settings)
+        {
+        }
+    }
+
+    public class RectIntTests : TypeTester<RectInt>
+    {
+        public static (RectInt deserialized, string serialized)[] representations = new[] {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=4930242)